antora-ui-camel/src/css/header.css

```
@@ -390,15 +410,45 @@ body {
   padding-right: 0.5rem;
 }
 
+#search_results div.result {
+  padding: 0.25rem;
+  pointer-events: none;
+}
+
+#search_results div.result:hover {
+  background: var(--color-smoke-50);
+}
+
+#search_results div.section {
+  display: flex;
+}
+
```

This was causing scrollbars to show

```suggestion

  #search_results dl {
    margin: 0;
  }
```

---

This doesn't remove scrollbars necessarily. Scrollbars only appear when the text covered by dl has height > 80vh which occurs in a few cases. Hence, I suggested using pagination to avoid scrollbars usage. However, for this PR, I wanted to just have a basic implementation for the search design layout.